open-budget-il/main.py

```diff
@@ -677,22 +680,23 @@ def get_query(self):
         index = search.Index(name="OpenBudget")
         results = []
         try:
-            searchQueryStringList = [queryString]
```

Comment:
Actually your query will return all BL and EN containing מדינה as well as all the BL from 2014  regardless of their title.

What we want are all the BL containing מדינה from 2014 and all EN containing מדינה